saltstack/base/salt/selinux/etc/selinux/cdp/salt-bootstrap/cdp-salt-bootstrap.te

```diff
@@ -0,0 +1,167 @@
+policy_module(cdp-salt-bootstrap, 1.0.0)
+
+########################################
+#
+# Declarations, domain setup
+#
+
+require {
+    type cdp_salt_bootstrap_t, cdp_salt_bootstrap_exec_t;
+}
+init_daemon_domain(cdp_salt_bootstrap_t, cdp_salt_bootstrap_exec_t)
+
+########################################
+#
+# cdp-salt-bootstrap local policy
+#
+allow cdp_salt_bootstrap_t self:fifo_file rw_fifo_file_perms;
+allow cdp_salt_bootstrap_t self:unix_stream_socket create_stream_socket_perms;
+
+require {
+    type cdp_salt_bootstrap_etc_t, cdp_salt_bootstrap_cert_t;
+}
+search_dirs_pattern(cdp_salt_bootstrap_t, cdp_salt_bootstrap_etc_t, cdp_salt_bootstrap_etc_t)
+search_dirs_pattern(cdp_salt_bootstrap_t, cdp_salt_bootstrap_cert_t, cdp_salt_bootstrap_cert_t)
+read_files_pattern(cdp_salt_bootstrap_t, cdp_salt_bootstrap_etc_t, cdp_salt_bootstrap_etc_t)
+read_files_pattern(cdp_salt_bootstrap_t, cdp_salt_bootstrap_cert_t, cdp_salt_bootstrap_cert_t)
+
+require {
+    type cdp_salt_bootstrap_log_t, var_log_t;
+}
+manage_dirs_pattern(cdp_salt_bootstrap_t, var_log_t, cdp_salt_bootstrap_log_t)
```

Comment:
I assume this line is not strictly necessary, since we don't have a separate directory for saltboot logs inside `/var/log/`, but I am fine with it staying here.

Reply:
Correct. This was originally included in the `sepolicy generate` output, and I did not want to remove it. :smile: